components/locid_transform/src/fallback/mod.rs

```diff
@@ -148,7 +148,11 @@ impl LocaleFallbacker {
             parents: crate::provider::Baked::SINGLETON_FALLBACK_PARENTS_V1,
             collation_supplement: Some(crate::provider::Baked::SINGLETON_FALLBACK_SUPPLEMENT_CO_V1),
         };
-        // Shitty covariance because the zeromaps confuse the compiler
+        // Safety: we're transmuting down from LocaleFallbackerBorrowed<'static> to LocaleFallbackerBorrowed<'a>
```

Comment:
wait, you think my comment wasn't accurate enough 😀 ?

Reply:
trying to make it understandable for people less familiar with this stuff

Reply:
I understand, this was meant as a light-hearted joke